OpenQA/Benchmark/Stopwatch.pm

```diff
@@ -262,13 +262,9 @@ Inspiration from my colleagues at L<http://www.nestoria.co.uk>
 
 =head1 COPYRIGHT
 
-Copyright (C) 2006 Edmund von der Burg. All rights reserved.
-
-This module is free software; you can redistribute it and/or modify it under
-the same terms as Perl itself. If it breaks you get to keep both pieces.
-
-THERE IS NO WARRANTY.
+Copyright 2006 Edmund von der Burg
 
+SPDX-License-Identifier Artistic-1.0 OR GPL-1.0-or-later
```

Comment:
the format is with `:` though

Reply:
thx, I did that wrong in my manual edits of the two "artistic" files. Corrected as well as the C/C++ files I missed. Updated PR